women-in-tech-resources/resources-list.md

```diff
@@ -33,16 +33,71 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 - [Moms In Tech - Facebook](https://www.facebook.com/groups/momsintechgroup/)
 
 
+### Meetups and Events for Women & Non-binary Engineers:
+- [Lesbians Who Tech](https://www.eventbrite.com/o/lesbians-who-tech-allies-5082754961)
+- [PyLadies](https://www.pyladies.com/)
+- [Women Who Code - Networks](https://www.womenwhocode.com/networks)
+- [Hacktoberfest](https://hacktoberfest.digitalocean.com/)
```

Comment:
Could you please move Hacktoberfest to Open Source and Hackathons? People have related events sometimes, but I wouldn’t describe it as a meetup.

Reply:
Sure